documentation/spring-boot-starter.html.haml

```diff
@@ -71,21 +71,29 @@ layout: base
 %p
   The <code>togglz-spring-boot-starter</code> will automatically trigger the Togglz Spring Boot auto configuration.
   It will create all necessary beans and in particular the Togglz <code>FeatureManager</code>.
-  You can <code>@Autowired</code> the <code>FeatureManager</code> into your application code, and then use
-  it to check if a feature is enabled, e.g.
+  You can inject the <code>FeatureManager</code> into your application code (using <a href="https://docs.spring.io/spring/docs/4.2.x/spring-framework-reference/html/beans.html#beans-constructor-injection" target="_blank">constructor injection</a>), and then use it to check if a feature is enabled, e.g.
 
 %pre(class="prettyprint lang-java")
   :escaped
-    @Autowired
-    private FeatureManager manager;
-
-    @RequestMapping("/")
-    public ResponseEntity<?> index() {
-        if (manager.isActive(HELLO_WORLD)) {
-             ...
-        }
+    @Controller
+    public class MyClass {
+      private FeatureManager manager;
+
+      public MyClass(Feature manager) {
```

Comment:
I guess `Feature` should be `FeatureManager`?

Reply:
Double-checked it several times to not run into a typo and what happens? A typo... 😞

Reply:
:laughing: